app/lib/controlled_work_mappings/cw1.yml

```diff
@@ -0,0 +1,495 @@
+# This file defines the mappings used to prepopulate a CW1 PDF form with the answers given by a user to CCQ's questions
+# combined with the output from CFE.
+
+# The mappings conform to the format described in ControlledWorkDocumentValueMappingService
+
+# "The client is directly or indirectly in receipt of S.4 or S.95 Asylum Support (Immigration and Asylum category work only):"
+- name: Go to question 2
+  type: boolean_radio
+  source: from_attribute
+  attribute: asylum_support
+  yes_value: "Yes_2"
```

Comment:
I was a bit confused about the yes_value/no_value entries, but I realised that these suffixes like "_2" were  probably used to identify where on a page these were appearing. As such I assume that all suffixes on yes/no answers need to be removed now.

Reply:
@willc-work sorry, I should probably have documented this better. For form radio buttons, each button in the set has its own value. To mark that button as selected you have to assign that value to the radio set. So in this case, if you want the 'Yes' button to be checked for the "Does the client receive asylum support?" question, you have to assign the value "Yes_2" to the set called "Go to question 2", because "Yes_2" happens to be the value associated with the "Yes" button in the "Go to question 2" set.